tests/test_core.py

```diff
@@ -46,6 +48,22 @@ def test_device_unicode(self):
             for device in devices:
                 print("device: %s" % device)
 
+    def test_device_failed_properties(self):
+        """
+        Makes sure printing a device doesn't crash.
+        """
+
+        with captured_output() as (out, err):
+            dev = mock.Mock()
+            dev.GetId = mock.Mock(return_value="id")
+            dev.GetState = mock.Mock(return_value=AudioDeviceState.Active)
+            store = mock.Mock()
+            store.GetCount = mock.Mock(return_value=3)
+            store.GetAt = mock.Mock(return_value="pk")
+            store.GetValue = mock.Mock(side_effect=_ctypes.COMError(None, None, None))
+            dev.OpenPropertyStore = mock.Mock(return_value=store)
+            audev = AudioUtilities.CreateDevice(dev)
```

Comment:
I think only this line needs to be in the `captured_output` context manager, correct?